Doc/library/venv.rst

```diff
@@ -281,9 +281,9 @@ subclass which installs setuptools and pip into a created virtual environment::
         This builder installs setuptools and pip so that you can pip or
         easy_install other packages into the created virtual environment.
 
-        :param nodist: If True, setuptools and pip are not installed into the
+        :param nodist: If true, setuptools and pip are not installed into the
```

Comment:
I approve this change.

Reply:
I believe the doc build is failing here because `:param nodist:` is being falsely recognized as a Sphinx role. This could be fixed by removing the first colon `:` from each of them. Would that be okay @vsajip?

```suggestion
        param nodist: If true, setuptools and pip are not installed into the
```

Doc build failure:
```
Warning, treated as error:
[library/venv:270] ":param" found in "    :param nodist: If true, setuptools and pip are not installed into the"
 ```